SimG4CMS/Tracker/interface/TkAccumulatingSensitiveDetector.h

```diff
@@ -46,56 +45,51 @@ public Observer<const BeginOfJob*>
     void fillHits(edm::PSimHitContainer&, const std::string&) override;
     void clearHits() override;
 
-    std::string type();
-
 private:
-    virtual void sendHit();
-    virtual void updateHit(G4Step *);
-    virtual bool newHit(G4Step *);
-    virtual bool closeHit(G4Step *);
-    virtual void createHit(G4Step *);
-    void checkExitPoint(Local3DPoint);
+    void createHit(const G4Step *);
+    void sendHit();
+    void updateHit(const G4Step *);
+    bool newHit   (const G4Step *);
+    bool closeHit (const G4Step *);
+
+    void checkExitPoint(const Local3DPoint&);
     void update(const BeginOfEvent *) override;
     void update(const BeginOfTrack *) override;
     void update(const BeginOfJob *) override;
-    Local3DPoint toOrcaRef(Local3DPoint ,G4VPhysicalVolume *);
-    int tofBin(float);
 
-    TrackingSlaveSD * slaveLowTof;
-    TrackingSlaveSD * slaveHighTof;
-    FrameRotation * myRotation;
-    UpdatablePSimHit * mySimHit;
-    std::string pname;
-    Local3DPoint globalEntryPoint;
-    Local3DPoint globalExitPoint;
+    Local3DPoint toOrcaRef(const Local3DPoint&);
```

Comment:
maybe enough years have gone by that we can rename (or remove as it seems unused) this function to something more descriptive?

Reply:
It is removed now.